### PR TITLE
feat: add event bus backend

### PR DESCRIPTION
**Pull request by Ian2012**
## Description

This PR creates a backend to emit the Open edX signal defined in https://github.com/openedx/openedx-events/pull/230.

## Testing instructions
- Install openedx-events from the mentioned PR
- Install event-tracking from this branch
- Add the following setting:
```python
EVENT_TRACKING_BACKENDS = {
    'xapi': {
        'ENGINE':  'eventtracking.backends.event_bus.EventBusRoutingBackend',
        'OPTIONS': { ## Not sure if this is needed
            'backend_name': 'xapi',
            'processors': [
                {
                }
            ],
     },
   },
}
```
- Listen to the emitted signal and verify the data is received correctly for all the events.

For reference, here is the consumer code:

```python
@receiver(TRACKING_EVENT_EMITTED)
def listen_for_tracking_event_emitted_event(sender, signal, **kwargs):
    """
    Prints `TRACKING_EVENT_EMITTED` events.
    """
    print("\n\n", kwargs, "\n\n")
```

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @Ian2012!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


This is currently a draft pull request. When it is ready for our review and all tests are green, click "Ready for Review", or remove "WIP" from the title, as appropriate.

**Comment by bmtcril** [comment]
Can you add some basic tests like the async backend has? I think this will also require openedx-events in base.in requirements now as well.

File: eventtracking/backends/event_bus.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    def send(self, event):
+        """Emit the Open edX public signal"""
+        tracking_data = TrackingLogData(
+            name = event.get('name'),
+            timestamp = event.get('timestamp'),
+            data = event.get('data'),
+            context = event.get('context')
+        )
+
```
Comment (mariajgrimaldi) [inline eventtracking/backends/event_bus.py:23]: You're missing this kind of code annotations here, you can find examples in edx-platform :)

**Comment by Ian2012** [comment]
@bmtcril There are some tests implemented now, it will fail till the openedx signal is merged

**Comment by mphilbrick211** [comment]
Hi @Ian2012 - checking in to see if you plan to pursue this pull request?

**Comment by mphilbrick211** [comment]
Closing this PR due to inactivity.

**Comment by openedx-webhooks** [comment]
@Ian2012 Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/event-tracking&entry.867055334=https://github.com/openedx/event-tracking/pull/225&entry.1484655318=https://github.com/Ian2012&entry.752974735=2023-06-02+18:41&entry.1917517419=2023-10-24+20:29&entry.2133058324=No) so we can improve your experience in the future.

**Comment by openedx-webhooks** [comment]
@Ian2012 Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/event-tracking&entry.867055334=https://github.com/openedx/event-tracking/pull/225&entry.1484655318=https://github.com/Ian2012&entry.752974735=2023-06-02+18:41&entry.1917517419=2023-10-24+20:29&entry.2133058324=No) so we can improve your experience in the future.

**Pull request closed without merging**